### PR TITLE
fix pycodestyle in tests modules forked

### PR DESCRIPTION
**Pull request by ZeroDayPoke**
multiple instances of != being used in place of is not

additional temporary fix to base_model tests (error occurs due to new base_model)

### CODE REVIEW

**Review by Oggal — COMMENTED** [review]
Want to investigate some of these things, but all in all looks like good changes!

File: tests/test_models/test_base_model.py [hunk 1]
```diff
@@ -82,15 +82,12 @@ def test_datetime_attributes(self):
...
         inst1 = BaseModel()
-        toc = datetime.now()
-        self.assertTrue(tic <= inst1.created_at <= toc)
+        post_create_time = datetime.now()
         time.sleep(1e-4)
-        tic = datetime.now()
         inst2 = BaseModel()
-        toc = datetime.now()
-        self.assertTrue(tic <= inst2.created_at <= toc)
+        self.assertTrue(pre_create_time <= post_create_time)
```
Comment (Oggal) [inline tests/test_models/test_base_model.py:90]: Need to duplicate this line and check that inst1.created_at is before post_create and after pre_create

Comment (Oggal) [reply tests/test_models/test_base_model.py:90]: may be best to do in multiple lines

File: tests/test_models/test_engine/test_db_storage.py [hunk 2]
```diff
@@ -68,6 +70,36 @@ def test_dbs_func_docstrings(self):
                             "{:s} method needs a docstring".format(func[0]))
 
 
+@unittest.skipIf(type(models.storage) == FileStorage, "Testing FileStorage")
+class TestStateDBInstances(unittest.TestCase):
+    """DBStorage State Tests"""
+    def tearDownClass():
```
Comment (Oggal) [inline tests/test_models/test_engine/test_db_storage.py:76]: snake_case_please

File: tests/test_models/test_engine/test_db_storage.py [hunk 3]
```diff
@@ -68,6 +70,36 @@ def test_dbs_func_docstrings(self):
...
+        """tidies up the tests removing storage objects"""
+        storage.hcf(State)
+
+    def setUp(self):
```
[the remark this answers is not in the document]
Comment (Oggal) [reply tests/test_models/test_engine/test_db_storage.py:80]: unless this is required for unittest module

File: models/engine/db_storage.py [hunk 4]
```diff
@@ -86,3 +88,11 @@ def get(self, cls, id):
     def count(self, cls=None):
         """returns count of objs in cls"""
         return (len(self.all(cls)))
+
+    def hcf(self, cls):
```
[the remark this answers is not in the document]
Comment (ZeroDayPoke) [reply models/engine/db_storage.py]: method name fix

**Pull request merged**